zgw/verzoek/src/main/kotlin/com/ritense/verzoek/VerzoekPluginEventListener.kt

```diff
@@ -195,6 +223,14 @@ open class VerzoekPluginEventListener(
         }
     }
 
+    private fun getJsonValueWithPointer(jsonNode: JsonNode, pointer: String): String? {
+        val valueNode = jsonNode.at(pointer)
+        if (valueNode.isMissingNode) {
+            throw NoSuchElementException("Path does not exist in the JSON: $pointer")
+        }
+        return valueNode.asText()
```

Comment:
- A missing node should be saved as `null`. This is also how its done for the `doc:` prefix
- `asText()` will save booleans and numbers as a String 

```suggestion
        return if (jsonNode == null || jsonNode.isMissingNode || jsonNode.isNull) {
            null
        } else if (jsonNode.isValueNode || jsonNode.isArray || jsonNode.isObject) {
            objectMapper.treeToValue(jsonNode, Object::class.java)
        } else {
            jsonNode.asText()
        }
```